app/views/records/show_fields/_abstract.html.erb

```diff
@@ -0,0 +1,5 @@
+<span itemprop="abstract">
+  <%= iconify_auto_link(record.abstract) %>
```

Comment:
@criedlberger Not quite sure I follow the code here. iconify_auto_link is to turn the text into a link: http://www.rubydoc.info/gems/sufia/1.3.0/SufiaHelper:iconify_auto_link
But I think we want to do html_safe on the text?